DigitalLearningSolutions.Data.Tests/Services/UserServiceTests.cs

```diff
@@ -458,5 +458,48 @@ public void NewEmailAddressIsValid_returns_true_for_delegate_only_with_unchanged
             A.CallTo(() => userDataService.GetAdminUserByEmailAddress(email)).MustNotHaveHappened();
             A.CallTo(() => userDataService.GetDelegateUsersByEmailAddress(email)).MustNotHaveHappened();
         }
+
+        [Test]
+        public void ResetFailedLoginCount_resets_count()
+        {
+            // Given
+            var adminUser = UserTestHelper.GetDefaultAdminUser(failedLoginCount: 4);
+            A.CallTo(() => userDataService.UpdateAdminUserFailedLoginCount(adminUser.Id, 0)).DoesNothing();

```

Comment:
Does this mock need to be set up?

Reply:
I'm a lot more used to strict mocking from previous clients, never remember which I can drop safely.